docs/release-notes/software-releases.md

```diff
@@ -19,6 +19,30 @@ None to report.
 None to report.
 ------>
 
+## February 7, 2022: File Entity Page and Open Access Variant Filtering​
+
+Data Platform - 1.105.3 - API 3.28.2
+
+#### New Features
+
+- A new **File Entity** page has been released that displays the metadata about a file. This page is accessible from the [File Repository](https://platform.icgc-argo.org/repository) by clicking on a File ID. This page includes:
+  - A file summary with a link to all files for the associated program.
+  - Data analysis Information, that include helpful links to documentation about the associated ARGO DNA pipeline workflow and version.
```

Comment:
@ciaranschutte can you please change this to `Data analysis information` (lowercase i)